modules/ossm-threescale-webassembly-module-configuration.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-configuration_{context}"]
+= The 3scale WebAssembly module configuration
+
+The `ServiceMeshExtension` custom resource spec provides the configuration that the `Proxy-WASM` module reads from.
+
+The spec is embedded in the host and read by the `Proxy-WASM` module. Typically, the configurations are in the _JSON_ file format for the modules to parse, however the `ServiceMeshExtension` resource can interpret the spec value as _YAML_ and convert it to _JSON_ for consumption by the module.
```

Comment:
I don't know that "JSON" or "YAML" need to be italicized or marked up at all (throughout).